com.unity.render-pipelines.core/ShaderLibrary/ScreenCoordOverride.hlsl

```diff
@@ -0,0 +1,24 @@
+#ifndef UNITY_SCREEN_COORD_OVERRIDE_INCLUDED
+#define UNITY_SCREEN_COORD_OVERRIDE_INCLUDED
+
+float2 ScreenCoordApplyScaleBias(float2 xy, float4 screenCoordScaleBias)
+{
+    return screenCoordScaleBias.zw + xy * screenCoordScaleBias.xy;
+}
+
+float2 ScreenCoordRemoveScaleBias(float2 xy, float4 screenCoordScaleBias)
+{
+    return (xy - screenCoordScaleBias.zw) / screenCoordScaleBias.xy;
+}
+
+#if defined(SCREEN_COORD_OVERRIDE)
+    #define SCREEN_COORD_APPLY_SCALEBIAS(xy)          ScreenCoordApplyScaleBias(xy, _ScreenCoordScaleBias)
+    #define SCREEN_COORD_REMOVE_SCALEBIAS(xy)         ScreenCoordRemoveScaleBias(xy, _ScreenCoordScaleBias)
+    #define SCREEN_SIZE_OVERRIDE                     _ScreenSizeOverride
+#else
+    #define SCREEN_COORD_APPLY_SCALEBIAS(xy)          xy
+    #define SCREEN_COORD_REMOVE_SCALEBIAS(xy)         xy
+    #define SCREEN_SIZE_OVERRIDE                     _ScreenSize
```

Comment:
Do you mean `#define SCREEN_SIZE_OVERRIDE _PostProcessScreenSize`?

Reply:
I notice ths comment
// SCREEN_SIZE_OVERRIDE defaults to _ScreenSize when SCREEN_COORD_OVERRIDE is not defined.
#ifdef SCREEN_COORD_OVERRIDE
but it is true qs zenzhang mention that there is this replacement: https://github.com/Unity-Technologies/Graphics/pull/6597/files#diff-d50957ca80ebaf9db3fbd56554efbd4b827572fa4449b5b2245e677c5fa004d9L133

so what's happen?

Reply:
and here we have a ifdef SCREEN_COORD_OVERRIDE
for _PostProcessScreenSize usage, so there is certainly a mistake
https://github.com/Unity-Technologies/Graphics/pull/6597/files#diff-d50957ca80ebaf9db3fbd56554efbd4b827572fa4449b5b2245e677c5fa004d9R202

Reply:
see https://github.com/Unity-Technologies/Graphics/pull/6597#issuecomment-1014956858